File: 백준/Bronze/2566. 최댓값/최댓값.py
arr = []
max = 0
x=0
y=0

for i in range(9):
  arr.append(list(map(int,input().split())))

for i in range(9):
  for j in range(9):
    if max < arr[i][j]:
      max = arr[i][j]
      x=i
      y=j

print(max)
print(x+1,y+1)

File: 백준/Bronze/2562. 최댓값/최댓값.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "예제.txt";
let input = fs.readFileSync(filePath).toString().trim().split("\n").map(Number);
let maxNum = Math.max(...input);

console.log(maxNum);
console.log(input.findIndex((data) => data === maxNum) + 1);
